ui/logviewer.html

```diff
@@ -109,6 +111,8 @@
          lv-log-lines="displayedLogLines">
     </div>
 
+    <th-notification-box></th-notification-box>
```

Comment:
@camd so I'm re-using the existing notification partial, rather than duplicating one for logviewer. I think this makes sense for now (?) unless we have to diverge logviewer's later for some reason.